core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -531,14 +531,32 @@ namespace ledger {
             });
         }
 
+        // This method is used to get a block height only used to know which
+        // update/fork (activated at certain block height) is used.
+        // This explains why we take LLONG_MAX is we have no block in DB for
+        // certain currency
+        // WARNING: please don't use this method if you need an accurate value
+        // of last block
+        static uint64_t getLastBlockFromDB(soci::session &sql, const std::string &currencyName) {
+            //Get last block from DB
+            auto lastBlock = BlockDatabaseHelper::getLastBlock(sql, currencyName);
+            //If we can not retrieve a last block for currency, we apply last BIP/update/fork
+            return lastBlock.hasValue() ? static_cast<uint64_t >(lastBlock->height) : LLONG_MAX;
```

Comment:
[OPT] Define a constant for this, something like HIGHEST_POSSIBLE_BLOCK

Reply:
As discussed, I have just updated comments, introducing a new constant will just bring more confusing (specially with a name like HIGHEST_POSSIBLE_BLOCK)